docs/DeveloperGuide.md

```diff
@@ -415,6 +415,37 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
       Use case ends.
 
+**Use case: Delete a tag from a person**
+
+**MSS**
+
+1. User requests to list persons
+2. Coach2K22 shows a list of persons
+3. User requests to remove an existing tag from a person
+4. Coach2k22 shows the updated details of the person
+
+   Use case ends.
+
+**Extensions**
+
+* 1a. The persons list is empty.
```

Comment:
Should we change 1a to 

* 2a. The list is empty.

  Use case ends. 

instead?